Reviews/340 - NodeRED - Casper Integration/1 - Full project delivery/README.md

```diff
@@ -0,0 +1,367 @@
+  get-transfers.js                     |   76.47 |       50 |      80 |   76.47 | 27-32
+ get-validators-info                   |   76.47 |       50 |      80 |   76.47 |
+  get-validators-info.js               |   76.47 |       50 |      80 |   76.47 | 26-31
+ module-bytes                          |   82.35 |      100 |     100 |   82.35 |
+  module-bytes.js                      |   82.35 |      100 |     100 |   82.35 | 24-28
+ put-deploy                            |   89.18 |       50 |      80 |   89.18 |
+  put-deploy.js                        |   89.18 |       50 |      80 |   89.18 | 69-72
+ stored-contract-by-hash               |   83.33 |      100 |     100 |   83.33 |
+  stored-contract-by-hash.js           |   83.33 |      100 |     100 |   83.33 | 29-33
+ stored-contract-by-name               |   83.33 |      100 |     100 |   83.33 |
+  stored-contract-by-name.js           |   83.33 |      100 |     100 |   83.33 | 31-35
+ stored-versioned-contract-by-hash     |   84.21 |    85.71 |     100 |   84.21 |
+  stored-versioned-contract-by-hash.js |   84.21 |    85.71 |     100 |   84.21 | 31-35
+ stored-versioned-contract-by-name     |   84.21 |    85.71 |     100 |   84.21 |
+  stored-versioned-contract-by-name.js |   84.21 |    85.71 |     100 |   84.21 | 30-34
+---------------------------------------|---------|----------|---------|---------|----------------------------------------------------
+
+```
+
+The reviewer add the package NYC to the project to get the code coverage of the project out of curiosity.  
+The tests and code coverage are impressive.
+
+Requirement | Finding
+------------ | -------------
+Unit Tests - At least one positive path test | PASS
+Unit Tests - At least one negative path test | PASS
+Unit Tests - Additional path tests | PASS
+
+# Documentation
+
+### Code Documentation
+
+The OP added documentation as the reviewer asked it.
+The documentation is now acceptable.
+
+Requirement | Finding
+------------ | -------------
+Code Documented | PASS
+
+### Project Documentation
+
+The reviewer observed that the README of the project has minimal general and usage documentation.
+The README could be improved to be more user-friendly for people that don't know the node-red software.
+However, the documentation inside each node is well documented. 
+
+Requirement | Finding
+------------ | -------------
+Usage Documented | PASS with notes
+
+## Overall Conclusion on Documentation
+
+The reviewer concludes that the project has sufficient documentation but the README could be improved. 
+
+# Open Source Practices
+
+## Licenses
+
+The Project is released under the MIT License
+
+Requirement | Finding
+------------ | -------------
+OSI-approved open source software license | PASS
+
+## Contribution Policies
+
+Pull requests and Issues are enabled. The repository doesnt contains a CONTRIBUTING and a SECURITY policy.
+
+Requirement | Finding
+------------ | -------------
+OSS contribution best practices | PASS
```

Comment:
I think a `PASS with Notes` would be better here since you have some notes to fix.

Reply:
It was a typo :) Fixed it